plugin/storage/integration/grpc_test.go

```diff
@@ -87,6 +145,31 @@ func (s *GRPCStorageIntegrationTestSuite) cleanUp() error {
 	return s.initialize()
 }
 
+type memoryStorePlugin struct {
```

Comment:
This is copied from `examples/memstore-plugin/main.go` and I'd like to avoid duplicate code but I don't know where to put this so it can be shared. Ideas most welcome.

Reply:
plugin/storage/grpc/memory/